### ISSUE TITLE
Cannot change accounts (e.g. from xbox to steam)

### ISSUE THREAD

**Issue by tteke** [ChiriVulpes/deepsight.gg#7]

**Comment by ChiriVulpes** [comment]
The unauthorise option in the settings menu doesn't work for this?

**Comment by tteke** [comment]
No, it does not sadly. Doesnt matter which account I authorise connection with, it always opens up my xbox characters (which I do not play at all). It does this even if I login with steam. On other softwares theres option after authorising to select platform (like steam, xbox, etc.)

**Comment by ChiriVulpes** [comment]
Ah, I see. I'll take a look, potentially a bug that exists cuz I don't have a multi platform account to test with. Thanks for the additional info!

**Comment by ChiriVulpes** [comment]
Looks like I just never added support for when you have memberships via multiple platforms. I *believe* based on a bit of experimentation on my end that if you enabled cross save it would make the API send deepsight your steam membership, which would allow it to work? But don't bother, I'll add support for switching which membership you're looking at anyway, for others in your situation. ETA the next couple days probably!

**Comment by ChiriVulpes** [comment]
Alright, I've just pushed support for both cross save (by default, it shows the currently cross saved platform's characters) and manually selecting a specific platform to show the characters from. It's in the Settings tab if with your setup you're still seeing your Xbox characters by default.

Let me know how it goes! I'm going to pre-emptively close this assuming it's good, but I'll reopen if there's any issues.
